### DISCUSSION TITLE
Blazor render modes

### DISCUSSION THREAD

**Discussion by yarseyah** [mrpmorris/Fluxor#458] [General]
This might be a silly thought, but how exactly does Fluxor fit into the new RenderModes concept in Blazor where some parts of the system may be running on the server, others on the client and some even migrating during a session.

Can a server-rendered item access the IStore? Would there be two, one on the server and one on the client? Can they sync?

I'm currently avoiding installing the beta/rc to keep my machine aligned to released versions, so at the moment I'm thinking of problems but not able to try out things, but has anyone else? Is this even possible?

**Comment by mrpmorris** [comment]
SSR requests have no state persisted between requests. You'd have to store it in a session or something across requests. It wouldn't be shared between client and server.
